layers/+lang/perl5/packages.el

```diff
@@ -78,7 +78,7 @@
           (if (nth 4 (syntax-ppss))
               'font-lock-comment-face
             (if (= (- (match-end 2) (match-beginning 2)) 1)
-                (if (eq (char-after (match-beginning 3)) ?{)
+                (if (eq (char-after (match-beginning 3)) ?{})
```

Comment:
Please help me to determine whether this modification is correct.
Without `}`, emacs complains about mismatching parentheses.

Reply:
Hi @lebensterben 

`?<char>` is a syntax which Emacs represents characters.
```emacs-lisp
(eq ?{ (string-to-char "{"))
;; => t

(char-to-string ?{)
;; => "{"
```

So this change may not be necessary.

Reply:
I have seen the `<` function confuse it about mismatching parentheses as well. This is a problem with the Lisp mode code, I believe, and not with this line of code. The `?` syntax is a bit surprising sometimes: for example, `? ` is a space character (and I had thought Lisp syntax was space-agnostic).

Reply:
Thanks.
`?C` is the special read syntax.
https://www.gnu.org/software/emacs/manual/html_node/elisp/Special-Read-Syntax.html#Special-Read-Syntax

I will fix this in a new commit later.
The closing parentheses is added by pareninfer I think.

Reply:
Fixed.